zerver/tests/test_bugdown.py

```diff
@@ -1347,28 +1347,52 @@ def test_default_code_block_language(self) -> None:
         self.assertEqual(realm.default_code_block_language, None)
         text = "```{}\nconsole.log('Hello World');\n```\n"
 
-        # Render without default language
-        msg_with_js_before = bugdown_convert(text.format('js'))
-        msg_with_python_before = bugdown_convert(text.format('python'))
-        msg_without_language_before = bugdown_convert(text.format(''))
-
-        # Render with default=javascript
+        # 01 - Render without default language
+        msg_with_js_01 = bugdown_convert(text.format('js'))
+        msg_with_python_01 = bugdown_convert(text.format('python'))
+        msg_without_language_01 = bugdown_convert(text.format(''))
+        msg_with_quote_01 = bugdown_convert(text.format('quote'))
+        msg_with_math_01 = bugdown_convert(text.format('math'))
```

Comment:
I think these would be better named without using the `01` type numbers.  E.g. just `msg_with_js` for these; 

and then the next block should be e.g. `msg_with_python_default_js`, `msg_without_language_default_js` , etc.  

I think the end result will be the comparisons at the end of the test are way more readable.